weave/conftest.py

```diff
@@ -84,10 +84,62 @@ def pytest_collection_modifyitems(config, items):
     items[:] = selected_items
 
 
+PYTEST_CURRENT_TEST_ENV_VAR = "PYTEST_CURRENT_TEST"
```

Comment:
where is this env var set?

Reply:
Automatically by pytest

Reply:
https://docs.pytest.org/en/latest/example/simple.html#pytest-current-test-environment-variable